selenium-system-test/src/test/scala/hts/pages/AuthorityWizardPage.scala

```diff
@@ -95,8 +95,10 @@ object AuthorityWizardPage extends Page {
   def setRedirect(url: String)(implicit driver: WebDriver): Unit =
     find(name("redirectionUrl")).foreach(_.underlying.sendKeys(url))
 
-  def setNino(nino: String)(implicit driver: WebDriver): Unit =
+  def setNino(nino: String)(implicit driver: WebDriver): Unit = {
+    println("NINO entered on auth wizard: " + nino)
```

Comment:
at this point the NINO hasn't actually been entered. It would be more accurate to do:
```scala
find(name("nino")).foreach{ x =>
 x.underlying.sendKeys(nino)
println("...")
}
```
(`x` is not a good name above, I just can't remember what the type of it is right now)